pkgs/by-name/li/libpqxx/package.nix

```diff
@@ -0,0 +1,56 @@
+{
+  fetchFromGitHub,
+  lib,
+  postgresql,
+  python3,
+  stdenv,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libpqxx";
+  version = "7.9.2";
+
+  src = fetchFromGitHub {
+    owner = "jtv";
+    repo = "libpqxx";
+    rev = version;
+    hash = "sha256-I5e0iqXlZqDOMa1PlnrxpcKt1c2mbnSbVQrpi1Gh25o=";
+  };
+
+  outputs = [
+    "dev"
+    "out"
+  ];
+
+  postPatch = ''
+    patchShebangs ./tools/splitconfig.py
+  '';
+
+  configureFlags = [
+    "--disable-documentation"
+    "--enable-shared"
+  ];
+
+  CXXFLAGS = [ "-std=c++17" ];
```

Comment:
```suggestion
```
I'm uncertain of this line. I've added it in the hope that `mapnik` wouldn't error.